jax/_src/util.py

```diff
@@ -584,3 +585,20 @@ def decorator(f):
           "Decorator got wrong type: @use_cpp_method(is_enabled: bool=True)"
       )
     return decorator
+
+
+_version_regex = re.compile(r"([0-9]+(?:\.[0-9]+)*)(?:(rc|dev).*)?")
+def _parse_version(v: str) -> tuple[int, ...]:
+  m = _version_regex.match(v)
+  if m is None:
+    raise ValueError(f"Unable to parse version '{v}'")
+  return tuple(int(x) for x in m.group(1).split('.'))
+
+def numpy_version():
+  return _parse_version(np.__version__)
+
+
+if numpy_version() >= (1, 25, 0):
+  ComplexWarning = np.exceptions.ComplexWarning
+else:
+  ComplexWarning = np.ComplexWarning
```

Comment:
What if we named this `NumpyComplexWarning` from the start to avoid the need to rename the imports?

Reply:
Sure! I renamed it.